src/vs/workbench/contrib/notebook/browser/view/notebookCellList.ts

```diff
@@ -917,6 +912,17 @@ export class NotebookCellList extends WorkbenchList<CellViewModel> implements ID
 				}
 				break;
 			case CellRevealPosition.Bottom:
+				if (partial) {
+					const lineHeight = this.viewModel?.layoutInfo?.fontInfo.lineHeight ?? 15;
+					const firstLineLocation = elementTop + lineHeight + 20;
```

Comment:
I'm not sure if this extra 20px padding buffer can be calculated more smartly, but it seems to work for all toolbar configurations.

Reply:
I think it's just the space between the top of the cell element, and the top of the first line within the cell.

Reply:
it looks like 6px from the padding-top of .cell-inner-container, 12px from positioning the first line within the cell.
2 more just to give some padding.

Reply:
padding top is `this.viewContext.notebookOptions.getLayoutConfiguration().cellTopMargin`, first line padding is `this.viewContext.notebookOptions.getLayoutConfiguration().editorTopPadding`